### ISSUE TITLE
Make a continuous build of master for use?

### ISSUE THREAD

**Issue by BennyHarv3** [terraform-providers/terraform-provider-vault#298]
Would it be possible to have a build of master or prerelease or some such available for general use? I understand that you can build and stick the binary in a folder on a machine for use or release it as a third-party plugin release, but it would be nice to just be able to grab the latest build with a normal `providers` block with the beta version.

In my particular use case, I'm looking to use a single resource that isn't in the latest official release, but there might be other reasons for making such a build available.

**Comment by lawliet89** [comment]
Basing off the instructions in the [GCP provider](https://github.com/terraform-providers/terraform-provider-google/blob/master/.github/CONTRIBUTING.md), you can do this yourself:

Check out the repository to your GOPATH:

```
$ mkdir -p $GOPATH/src/github.com/terraform-providers; cd $GOPATH/src/github.com/terraform-providers
$ git clone git@github.com:terraform-providers/terraform-provider-vault
```

Build the plugin: `make build`.

Create a symbolic link:

```
ln -s $GOPATH/bin/terraform-provider-vault ~/.terraform.d/plugins/terraform-provider-vault
```

**Comment by BennyHarv3** [comment]
Yup, and I have successfully built it myself, but I'm asking for a nightly or prerelease build hosted like the normal releases. It would make using nightly or prerelease builds easier, particularly for CI environments where you might not be able to build the provider in the environment but could freely download the built provider.
